### ISSUE TITLE
problem with <br>\n

### ISSUE THREAD

**Issue by minhna** [jpuri/html-to-draftjs#19]
I use `react-draft-wysiwyg` editor, to save data to database, I use `draftjs-to-html` to convert content to HTML:
```
const rawContentState = convertToRaw(this.state.editorState.getCurrentContent());
const rawHTML = draftToHtml(rawContentState);
```
then use `html-to-draftjs` to convert it back to draftjs contentState
```
const blocksFromHtml = htmlToDraft(rawHTML);
const contentBlocks = blocksFromHtml.contentBlocks;
const contentState = ContentState.createFromBlockArray(contentBlocks);
```
The problem is it added more one empty line at `<br>\n`.
for example, the original editor looks like this:
```
One line
Other line

Other paragraph
```
become this:
```
One line

Other line

Other paragraph
```

versions:
"draft-js": "^0.10.1",
"html-to-draftjs": "0.1.0-beta7",
"react-draft-wysiwyg": "^1.10.0",
"draftjs-to-html":"^0.7.2"

Thank you.

**Comment by jpuri** [comment]
Hi @minhna : Sorry for the delay. I could replicate the issue. Looking into fixing it now.

**Comment by jpuri** [comment]
Issue has been fixed in `html-to-draftjs@0.1.0-beta10`.
